src/clj/puppetlabs/services/jruby/jruby_core.clj

```diff
@@ -142,43 +142,43 @@
 ;;; Public
 
 (schema/defn ^:always-validate
-  initialize-config :- jruby-schemas/JRubyPuppetConfig
+  initialize-config :- jruby-schemas/JRubyConfig
   [config :- {schema/Keyword schema/Any}]
-  (-> (get-in config [:jruby-puppet])
+  (-> (get-in config [:jruby])
```

Comment:
We're going to have to be careful with this one, right?  If we change this, then it will be a breaking change in Puppet Server's config files?